Services/SelectedStopPointManager.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace CanalTP\MttBundle\Services;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\Serializer\Serializer;
```

Comment:
Can you use use JMS\Serializer\SerializerInterface please? as we use Jms serializer instead of symfony one.

You will need to require "jms/serializer-bundle": "0.12.0" in your composer json